src/components/button/button.css

```diff
@@ -0,0 +1,12 @@
+.parentButton{
+    background-image: linear-gradient(to right bottom, var(--sun-glow), var(--orange), var(--indian-red), var(--fuchsia-rose));
+    height: 85px;
+    width: 350px;
+    border-radius: 20px;
+    text-align: center;
+    color: var(--baby-powder);
+    font-family: 'DM Sans';
+    font-size: 30px;
+    border: none;
+    font-weight: bold;
+}
```

Comment:
could you group the css styles according to their functionality?
for example groups could be :
background-image
height/width
border-related styles (border-radius, border)
text-related styles (color, text-align...)

So it should look something like "
.parentButton{
   background-image: ...
   
   height:...
   width:...

    border:...
    border-radius:...
   .
   .
   .
}